### PR TITLE
Add replication related use cases.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Partly addresses issue #21, along with the companion PR #94.

### CODE REVIEW

**Comment by OR13** [comment]
open for 14 days, we should merge this in 24 hours.

File: use_cases.md [hunk 1]
```diff
@@ -44,6 +44,48 @@ synchronize between each other so data is up to date in both places regardless
...
+Alice sets up replication to a backup instance so that she does not lose her
+data in case her primary instance is damaged/lost etc.
+
+* Replication for purposes of data backup
+* Either involves periodic full-sync replications, or a combination of realtime
+  replication and full-syncs.
+* Typically involves unidirectional replication (from an "active" source
+  instance to the target backup instance).
+
+#### 1.4.2 Continuous Replication / Replication on Reconnect
```
Comment (tplooker) [inline use_cases.md:58]: Is this really the same `Geographic availability`? 1.4.1 and 1.4.3 feel like use cases for Alice where as this one feels like how 1.4.3 is achieved

File: use_cases.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ when I have explicitly given consent for each item.
 I want to be able to revoke the access of others at any time. When sharing data,
 I can include an expiration date for the access to my data by a third-party.
 
-### ​1.4.​ Store the same data in more than one place
+### 1.4 Replication - Store the same data in more than one place
```
Comment (tplooker) [inline use_cases.md:37]: Given the discussion on the WG is "Preventing Vendor Lock In" another valid usecase?

Comment (OR13) [reply use_cases.md:37]: I think so, I think it applies to both replication and synchornization, and in general data portability.

**Comment by OR13** [comment]
I'm merging this, please open subsequent issues to continue the discussions if needed.

**Pull request merged**